src/app/shared/component/song-table/song-table.component.css

```diff
@@ -1,3 +1,20 @@
+.card-body {
+    padding: 0;
+}
+
+table {
+    margin: 0;
+}
+
+table * {
```

Comment:
Might be able to get something similar using the table-sm class here. https://getbootstrap.com/docs/4.0/content/tables/#small-table